packages/grid/x-data-grid/src/hooks/features/clipboard/useGridClipboard.ts

```diff
@@ -25,6 +25,20 @@ function writeToClipboardPolyfill(data: string) {
   }
 }
 
+function hasNativeSelection(element: Element) {
+  if (window.getSelection()?.toString() !== '') {
+    return true;
+  }
+
+  if (element instanceof HTMLInputElement || element instanceof HTMLTextAreaElement) {
```

Comment:
I don't think that we should use `instanceof HTMLXXXElement` This doesn't work cross frames. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_realms

Reply:
Thanks, I did not know that!
I used `instanceof` only to make Typescript happy. I'll use type assertion then.